src/main/java/de/retest/surili/commons/util/BigListOfNaughtyStrings.java

```diff
@@ -0,0 +1,36 @@
+package de.retest.surili.commons.util;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.UncheckedIOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class BigListOfNaughtyStrings {
+	private static final String FILENAME = "blns.json";
+	private static final List<String> BIG_LIST_OF_NAUGHTY_STRINGS = loadBigListOfNaughtStrings();
+
+	private static List<String> loadBigListOfNaughtStrings() {
+		final List<String> bigListOfNaughtStrings = new ArrayList<>();
+
+		try ( final InputStream url = BigListOfNaughtyStrings.class.getClassLoader().getResourceAsStream( FILENAME ) ) {
+			final ObjectMapper mapper = new ObjectMapper();
+			final List<String> strings = mapper.readValue( url,
+					mapper.getTypeFactory().constructCollectionType( List.class, String.class ) );
+			bigListOfNaughtStrings.addAll( strings );
+		} catch ( final IOException e ) {
+			throw new UncheckedIOException( "Exception reading '" + FILENAME + "'.", e );
+		}
+		return bigListOfNaughtStrings;
+	}
+
+	public static List<String> getBigListOfNaughtStrings() {
+		return BIG_LIST_OF_NAUGHTY_STRINGS;
```

Comment:
To make sure a caller doesn't modify it:

```suggestion
		return Collections.unmodifiableList( BIG_LIST_OF_NAUGHTY_STRINGS );
```

Reply:
Good catch.